Engine/entry1.c

```diff
@@ -166,6 +166,7 @@ OENTRY opcodlst_1[] = {
   { "init.a", S(ASSIGNM),0,       "mmmmmmmmmmmmmmmmmmmmmmmm", "m", mainit },
   { "nstrnum",   S(IREF_NUM),0,    "i",    ":InstrDef;",   (SUBR) get_instr_num },
   { "init.instr", S(ASSIGN) ,0,  ":InstrDef;", ":InstrDef;", (SUBR) copyVarGenericInit},
+  { "floatsize", S(ASSIGN) ,0, "i", "", myflt_size },
```

Comment:
Just a note, my first reaction to this was why not a variable? But I think I'm 50/50 on it as an opcode and seems fine.

Reply:
I thought opcode was easier to implement.